server/routes/addPrisoner/prisonerDetailsController.test.ts

```diff
@@ -272,22 +272,23 @@ describe('Prisoner details', () => {
           })
       })
 
+      const tomorrow = addDays(new Date(), 1)
```

Comment:
Might be simpler just to set the year to 2066 or something like that as I think it's just checking for a DOB in the future and we're not going to reach 2066...well, hopefully 😂

Reply:
Yours is technically better as it'll never fail!